src/FsReveal/template.html

```diff
@@ -34,6 +34,14 @@
     <!--[if lt IE 9]>
     <script src="lib/js/html5shiv.js"></script>
     <![endif]-->
+    <script language="javascript" type="text/javascript">
+        function init()
+        {
+            websocket = new WebSocket("ws://localhost:8083/websocket");
```

Comment:
It is already hard coded here https://github.com/fsprojects/FsReveal/blob/master/build.fsx#L90
It could be made configurable but it would be considerable more complicated.